base/accumulate.jl

```diff
@@ -85,6 +85,26 @@ julia> cumsum(a, dims=2)
  1  3   6
  4  9  15
 ```
+
+Note the important difference between `cumsum(A)` and `accumulate(+, A)` for
```

Comment:
Could also use a `!!! note` style here as seen in, for example [this part of the manual](https://docs.julialang.org/en/latest/manual/calling-c-and-fortran-code/)